xds/src/main/java/io/grpc/xds/MessagePrinter.java

```diff
@@ -48,6 +50,8 @@
                 .HttpConnectionManager.getDescriptor())
             .add(HTTPFault.getDescriptor())
             .add(io.envoyproxy.envoy.config.filter.http.fault.v2.HTTPFault.getDescriptor())
+            .add(RBAC.getDescriptor())
+            .add(RBACPerRoute.getDescriptor())
```

Comment:
I'm wondering if we only register `RBACPerRoute`, will `anyMessage.unpack(RBAC.class)` be pretty-printed because `RBACPerRoute` contains a field of `RBAC` type.

Reply:
Yes. This is transitive. The list here would be _much_ longer if it weren't.

I believe it is actually transitive _per file_, so we can actually be pulling in a lot more simply because certain messages happened to be in the same file. But we aren't knowingly depending on that at least.